Pod/Classes/JNKeychain.m

```diff
@@ -125,31 +125,42 @@ + (id)loadValueForKey:(NSString *)key
 
 + (NSString *)getBundleSeedIdentifier
 {
-    static NSString *bundleSeedID = nil;
-    static dispatch_once_t _once;
-    dispatch_once(&_once, ^{
-        NSDictionary *query = @{
-                                (__bridge id)kSecClass: (__bridge NSString *)kSecClassGenericPassword,
-                                (__bridge id)kSecAttrAccount: @"bundleSeedID",
-                                (__bridge id)kSecAttrService: @"",
-                                (__bridge id)kSecReturnAttributes: (__bridge id)kCFBooleanTrue
-                                };
-        CFDictionaryRef result = nil;
-        OSStatus status = SecItemCopyMatching((__bridge CFDictionaryRef)query, (CFTypeRef *)&result);
-        if (status == errSecItemNotFound)
-        {
-            status = SecItemAdd((__bridge CFDictionaryRef)query, (CFTypeRef *)&result);
-        }
-        if (status == errSecSuccess)
-        {
-            NSString *accessGroup = [(__bridge NSDictionary *)result objectForKey:(__bridge NSString *)kSecAttrAccessGroup];
-            NSArray *components = [accessGroup componentsSeparatedByString:@"."];
-            bundleSeedID = [[components objectEnumerator] nextObject];
```

Comment:
Without making a copy of bundleSeedID we are referencing the value of an internal data structure which is eligible for releasing.